### DISCUSSION TITLE
Welcome to spline-ui Discussions!

### DISCUSSION THREAD

**Discussion by wajda** [AbsaOSS/spline-ui#144] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Salatich** [comment]
Hi folks, hi @wajda 
Do you know is it possible in spline-ui to see the execution time of ExecutionEvent / ExecutionPlan? For instance, I can see durationNs in ExecutionEvent but can't find anything in the UI

**Comment by wajda** [comment]
It's called "elapsed time" in the UI.
![image](https://user-images.githubusercontent.com/795479/185711501-aedd22af-17f3-484a-a533-999adaa4c0c4.png)
